llama-index-integrations/embeddings/llama-index-embeddings-huggingface-openvino/llama_index/embeddings/huggingface_openvino/base.py

```diff
@@ -0,0 +1,181 @@
+from typing import Any, List, Optional, Dict
+
+from llama_index.core.base.embeddings.base import (
+    DEFAULT_EMBED_BATCH_SIZE,
+    BaseEmbedding,
+)
+from llama_index.core.bridge.pydantic import Field, PrivateAttr
+from llama_index.core.callbacks import CallbackManager
+from llama_index.embeddings.huggingface.utils import format_query, format_text
```

Comment:
I think you're missing the `llama-index-embeddings-huggingface` dependency in `pyproject.toml`